js/main.js

```diff
@@ -21,23 +21,88 @@ function getData() {
 }
 getData();
 
-var likedThought = document.querySelector('#form-input');
 var text;
 var authorName;
 
-function handleClick(event) {
+function handleLike(event) {
   event.preventDefault();
 
   var savedQuote = {
     quote: text,
-    author: authorName
+    author: authorName,
+    entryId: data.nextEntryId
   };
 
   data.nextEntryId++;
-  likedThought.reset();
-  data.entries.unshift(likedThought);
-  ul.prepend(savedQuote);
+  // likedThought.reset();
+  data.quotations.unshift(savedQuote);
+  renderLiked(savedQuote);
+  viewSwap(savedQuote);
+}
 
+if ($button != null) {
+  $button.addEventListener('click', handleLike);
 }
 
-likedThought.addEventListener('click', handleClick);
+function renderLiked(entry) {
+  var card = document.createElement('p');
+  // card.className('card');
+  card.setAttribute('class', 'card');
+
+  var text = document.createElement('p');
+  text.textContent = entry.text;
+
+  var person = document.createElement('p');
+  person.textContent = entry.person;
+
+  return card;
+}
+
+var navLiked = document.querySelector('.nav-liked-thoughts');
+navLiked.addEventListener('click', viewSwap);
+
+function viewSwap(savedQuote) {
```

Comment:
This function name is inaccurate, it's not swapping views, it's generating the list of liked quotes.